sdcm/utils/aws_builder.py

```diff
@@ -26,12 +29,81 @@
 
 LOGGER = logging.getLogger(__name__)
 
+ASG_CONFIG_FORMAT = """
+import com.amazonaws.services.ec2.model.InstanceType
+import hudson.plugins.sshslaves.SSHConnector
+import hudson.plugins.sshslaves.verifiers.NonVerifyingKeyVerificationStrategy
+import hudson.model.*
+import com.amazon.jenkins.ec2fleet.EC2FleetCloud
+import jenkins.model.Jenkins
+
+// just modify this config other code just logic
+config = [
+    name: "{name}",
+    region: "{region_name}",
+    fleetId: "{asg_id}",
+    idleMinutes: 10,
+    minSize: {min_size},
+    maxSize: {max_size},
+    numExecutors: 1,
```

Comment:
do we want 1 to 1 mapping? wouldn't we have too many builder in this case?

Reply:
I can play with this number, but I'm not sure how high I can really set it. 
we can start with 30 and see how it's works

Reply:
Do you mean 30 as the maxSize?
I'm sure we need more than that at some points.

Reply:
No, 30 executers on each instance,

The problem is that artifacts might accumulate and I'm not sure 30 artifacts tests running at the same time on 2cpu would get very far...

Reply:
Let's start with 4 and see how it works.